src/sentry/ingest/userreport.py

```diff
@@ -38,8 +38,8 @@ def save_userreport(project, report, start_time=None):
         if event.datetime < start_time - timedelta(minutes=30):
             raise Conflict("Feedback for this event cannot be modified.")
 
-        report["environment"] = event.get_environment()
-        report["group"] = event.group
+        report["environment_id"] = event.get_environment().id
+        report["group_id"] = event.group.id
```

Comment:
This could raise when we call this from the ingest consumer; access `group_id` instead.

```suggestion
        report["group_id"] = event.group_id
```